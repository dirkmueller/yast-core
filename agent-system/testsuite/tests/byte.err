[Interpreter] tests/byte.ycp:16 Read (.byte, "tests/not-here.data") failed: No such file or directory
