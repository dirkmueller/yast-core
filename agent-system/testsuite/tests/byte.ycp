{
    return Read (.byte, "tests/data1.read");
}

{
    return Read (.byte, "tests/data2.read");
}

{
    byteblock data = Read (.byte, "tests/data2.read");
    return Write (.byte, "tmp.write.byte", data);
}

{
    # this must produce a error in the log
    return Read (.byte, "tests/not-here.data");
}

{
    # this must not produce a error in the log
    return Read (.byte, [ "tests/not-here.data", #[ff] ]);
}
