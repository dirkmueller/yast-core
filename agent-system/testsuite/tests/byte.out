(#[])
(#[536F667477617265206973206C696B65207365782E2049742773206265747465 72207768656E206974277320667265652E0A])
(true)
(nil)
(#[FF])
